app/scripts/app-init.js

```diff
@@ -111,14 +108,20 @@ function importAllScripts() {
   }
 }
 
+// Ref: https://stackoverflow.com/questions/66406672/chrome-extension-mv3-modularize-service-worker-js-file
 // eslint-disable-next-line no-undef
 self.addEventListener('install', importAllScripts);
 
 /*
- * Message event listener below loads script if they are no longer available.
+ * A keepalive message receiver to avoid Service Worker getting shut down due to inactivity.
+ * UI sends the message periodically.
```

Comment:
```suggestion
 * A keepalive message listener to prevent Service Worker getting shut down due to inactivity.
 * UI sends the message periodically, in a setInterval. 
 * Chrome will revive the service worker if it was shut down, whenever a new message is sent, but only if a listener was defined here.
```